src/platforms/javascript/common/sourcemaps/tools/webpack.mdx

```diff
@@ -88,6 +90,33 @@ module.exports = {
 };
 ```
 
+</PlatformSection>
+
+<PlatformSection supported={["javascript.nextjs"]}>
+
+```javascript {filename:next.config.js}
+const { withSentryConfig } = require("@sentry/nextjs");
+
+const moduleExports = {
+  output: {
+    path: path.join(__dirname, "dist"),
+    filename: "[name].js",
+    sourceMapFilename: "[name].js.map",
+  },
+  // Other configuration.
+};
+
+const SentryWebpackPluginOptions = {
+  // Additional config options for the Sentry Webpack plugin.
+};
+
+module.exports = withSentryConfig(moduleExports, SentryWebpackPluginOptions);
+```
```

Comment:
`output` is webpack config, not nextjs config.

```suggestion
```javascript {filename:next.config.js}
const { withSentryConfig } = require("@sentry/nextjs");

const moduleExports = {
  webpack: (config, options) => {
    config.output = {
      ...config.output,
      path: path.join(__dirname, "dist"),
      filename: "[name].js",
      sourceMapFilename: "[name].js.map",
    };
    return config;
  },
  // Other nextjs configuration
};

const SentryWebpackPluginOptions = {
  // Config options for the Sentry Webpack plugin
};

module.exports = withSentryConfig(moduleExports, SentryWebpackPluginOptions);
```
```